lang/go/eval/symbol/struct.go

```diff
@@ -73,6 +74,21 @@ type FieldSymbol struct {
 	Value   Symbol `ko:"name=value"`
 }
 
+func disassembleFieldSymbolsToGo(span *model.Span, fields FieldSymbols) (map[string]interface{}, error) {
```

Comment:
See comment for Series. For structs you can rebuilt the exact structure
using the Go reflect MakeStruct.